shiny/api-examples/layout_column_wrap/app-express.py

```diff
@@ -0,0 +1,18 @@
+import shiny
+from shiny.express import ui
+
+a_card = shiny.ui.card("A simple card")
```

Comment:
```suggestion
from shiny.express import ui

with ui.hold() as a_card:
    with ui.card():
        "A simple card"
```

Reply:
oh thanks, sorry I just pushed work I had started on a week ago

Reply:
a lot has changed in a week!